scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/JobState.java

```diff
@@ -288,7 +288,7 @@ private TaskStatesPage getTaskStatesPage(int offset, int limit, List<TaskState>
      * To get the list of available tags in a job.
      * @return the list of tags.
      */
-    public List<String> getTags() {
+    public List<String> getTaskTags() {
```

Comment:
instead of renaming this existing method, which returns indeed legacy tags assigned to tasks, I would suggest that you call your new method and associated attribute getWorkflowTags